trigger/mqtt/README.md

```diff
@@ -0,0 +1,61 @@
+# MQTT
+This activity allows you to listen to message on Mqtt Queue.
+
+## Installation
+
+### Flogo CLI
+```bash
+flogo install github.com/project-flogo/edge-contrib/trigger/mqtt
+```
+
+## Configuration
+
+### Settings:
+| Name      | Type   | Description
+| :---      | :---   | :---
+| broker    | string | 	The broker URL
+| id | string | The id of client 
+| user | string | The name of the user
+| password | string | The password of the user
+| store | string | The path containing certificates
+
+### Handler Settings
+| Name      | Type   | Description
+| :---      | :---   | :---
+| cleansess | string | Clean sesssion
+| topic | string | The topic to publish to
```

Comment:
isn't this the top to listen on, probably should check this description in other places